### ISSUE TITLE
standard_real_sweep in phypy doesn't stop at target value

### ISSUE THREAD

**Issue by a1650078** [janverschelde/PHCpack#27]
I have been simulating mechanical linkages in phcpy and found a situation where phcpy no longer gives the solutions I am expecting. In my code, I am using sweeps to efficiently move from one solution of the linkage to a nearby solution. For example, I would like the angle of one joint to move from 10 degrees to 11. I am trying to do this by using a standard_real_sweep with an equation such as `(1-s) * 10 + s * 11 - angle ` (well actually I am using complex numbers for angles but you get the idea). This doesn't work, and I have reduced my code to the following modification of an example use of the library

```
from phcpy.sweepers import standard_real_sweep as sweep

circle = ['x^2 + y^2 - 1;', 'y*(1-s) + (y-2)*s;' ]

from phcpy.solutions import make_solution as makesol
first = makesol(['x', 'y', 's'], [1, 0, 0])
second = makesol(['x', 'y', 's'], [-1, 0, 0])

startsols = [first]

from phcpy.sweepers import standard_real_sweep as sweep
newsols = sweep(circle, startsols, par='s', start=0.0, target=0.1)

print(newsols[0])
```

Here I would expect to be given the solution (for x and y) when s = 0.1, but instead I am given the singular solution at s = 0.5. Why didn't the sweep stop earlier?

**Comment by janverschelde** [comment]
Thanks for reporting this.  I will try to fix it in one of the next releases of phcpy.

**Comment by janverschelde** [comment]
This should be fixed now in the latest version 1.0.3 of phcpy.
Therefore, I am closing this issue.
